### PR TITLE
Loading status for the header

### PR DESCRIPTION
**Pull request by jessieyu777**
- migrate elements for the loading status from gitness to playground
- make the design aligned with dropdown in UX for the component consistency 
- only rendering data in the playground and still left the logic in the gitness
- loading status
Screenshot 2024-09-12 at 4 05 52 PM
- loaded status
Screenshot 2024-09-12 at 4 12 50 PM

### CODE REVIEW

File: packages/playground/src/pages/create-pipeline-page.tsx
Comment (jessieyu777) [inline packages/playground/src/pages/create-pipeline-page.tsx:1]: fix the bug from rendering error

File: packages/playground/src/index.ts
Comment (jessieyu777) [inline packages/playground/src/index.ts:1]: export a loading status component

File: packages/playground/src/components/layout/top-bar-loading.tsx
Comment (jessieyu777) [inline packages/playground/src/components/layout/top-bar-loading.tsx:1]: migrate elements of the header from gitness package to the componet in the playground package

File: apps/gitness/src/components/Header.tsx
Comment (jessieyu777) [inline apps/gitness/src/components/Header.tsx:1]: import loading component in the gitness

**Comment by vardanbansal-harness** [comment]
Closing as not needed anymore.

**Pull request closed without merging**